zerver/openapi/zulip.yaml

```diff
@@ -14343,6 +14350,13 @@ paths:
                           Present if `realm` is present in `fetch_event_types`.
 
                           The name of the organization, used in login pages etc.
+                      realm_require_unique_names:
+                        type: boolean
+                        description: |
+                          Indicates whether the organization enforces users to have
+                          same full_names (if set to true) or not.
```

Comment:
We should spell out more precisely. 

> Indicates whether the organization is configured to require users to have unique full names. If `true`, the server will reject attempts to create a new user, or change the name of an existing user, where doing so would lead to two users whose names are identical modulo case and unicode normalization.